quilt3distribute/tests/data/zzz_generate_fake_dataset.py

```diff
@@ -68,6 +69,22 @@ def main():
     data = pd.DataFrame(rows)
     data.to_csv((BASE_PATH / "example.csv"), index=False)
 
+    ###############################################################################
+
+    # Create fake repeated dataset
+    cell_ids = [i for i in range(1, 10)]
+    source_read_paths_1 = [(BASE_PATH / "fake_images" / "3d" / "1.tiff").resolve() for i in range(4)]
```

Comment:
Assuming this script is being used as a test. Maybe add a path with a different path but identical filename. Assuming the files are different but have the same name this seems like a case where you wouldn't want to collapse them to one thing.

Reply:
I think you commented right before I pushed that commit. Line 89 now constructs exactly that dataset for testing with an additional test being added to the `test_dataset.py` file.